packages/tools/devtools/devtools-browser-extension/src/content/ContentScript.ts

```diff
@@ -9,8 +9,9 @@ import {
 	isDevtoolsMessage,
 } from "@fluid-experimental/devtools-core";
 
-import { extensionMessageSource, relayMessageToPort, relayMessageToWindow } from "../messaging";
-import { browser } from "../utilities";
+import { extensionMessageSource, relayMessageToPort } from "../messaging";
+// eslint-disable-next-line import/no-internal-modules
+import { browser, window } from "../utilities/Globals";
```

Comment:
Has to import directly from the module in order for the stubbing to work out correctly. I'm going to move this module to the root of the package so we don't have to do this drilling.